client/webserver/site/src/js/registry.ts

```diff
@@ -314,6 +315,12 @@ export interface OrderNote extends CoreNote {
   order: Order
 }
 
+export interface RecentMatch {
+  Rate: number
+  Qty: number
+  Age: Date
```

Comment:
Why are these upper-case? Add JSON tags or do type conversion as needed to fix this.